### PR TITLE
Added "select" component with PatternFly4 styles

### PR DESCRIPTION
**Pull request by akurinnoy**
Signed-off-by: Oleksii Kurinnyi <okurinny@redhat.com>

### What does this PR do?

This PR adds new "select" component based on PatternFly4 styles.

**Screenshot**
Screenshot 2020-04-07 at 18 54 06

### What issues does this PR fix or reference?

resolves https://github.com/eclipse/che/issues/16524

### CODE REVIEW

File: src/components/che-pf-widget/select/che-pf-select.directive.ts [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+    '$document',
+    '$sce',
+    'randomSvc',
+  ];
+
+  bindToController = true;
+  controller = 'ChePfSelectController';
+  controllerAs = 'ctrl';
+  templateUrl = 'components/che-pf-widget/select/che-pf-select.html';
+  // replace = true;
```
Comment (olexii4) [inline src/components/che-pf-widget/select/che-pf-select.directive.ts:84]: Could you remove this line ```// replace = true;``` ?

Comment (akurinnoy) [reply src/components/che-pf-widget/select/che-pf-select.directive.ts:84]: sure

**Comment by olexii4** [comment]
It will be nice to add a simple example of using this new widget into demo-components.html

**Comment by olexii4** [comment]
LGTM

**Pull request merged**